### DISCUSSION TITLE
Help me out

### DISCUSSION THREAD

**Discussion by ManojMk01** [Bibleup/bibleup#49] [Q&A] [answered]
Hello, About this wordpress plugin: https://wordpress.org/plugins/bibleup/ I want to change the footer text on the popup that says "BibleUp". It seems that this comes from .js CDN file from what i could tell. Do you know any way to replace this text? And still have the plugin working fine?

**Comment by Bukunmikuti** [comment, marked as answer]
Hi @ManojMk01 👋

The "BibleUp 📖💡" footer text only serve as a little promotion for BibleUp. It is mentioned in the documentation [here](https://bibleup.netlify.app/docs/customisation/#credit-bibleup)

> If you enjoyed using BibleUp, we plead that you leave the popup footer (containing BibleUp 📖💡) untampered with or without obstruction. The text does not link to our website or any affiliate website, it only serves as a little promotion for this tool.

However, if you would still like to remove it, there are basically two options:

1. **Modify the source code**
To do this you would have to clone the repo, modify the code (the text can be found in the [construct-popup.ts file](https://github.com/bibleup/bibleup/blob/main/bibleup/construct-popup.ts#L83C11-L83C11)) then build a minified file without the footer text. After that, you can upload the file to a CDN, and modify the WordPress plugin file to fetch from your new CDN link instead of BibleUp's. 
Note that this means you will be using your BibleUp file, without access to updates.

2. **Load your own plugin**
If you don't want to use a CDN and want a direct approach, you will have to clone and change the code of the [WordPress plugin](https://github.com/bibleup/wordpress) to use a local BibleUp file without the footer text, then you can install the plugin manually. You need to know how to develop a WordPress plugin to do this, I can help out if you have any issues.

I can also help out by providing a BibleUp build without the footer text but you will have to complete the other steps yourself. Note that for any method you choose, you will miss out on automatic feature updates and bug fixes because you will be using your own modified file.

**Reply by ManojMk01** [reply]
Thank you so much for your detailed explanation and the options provided. I genuinely appreciate your willingness to help and guide me through this process. Your assistance is invaluable. I'm definitely interested in obtaining a BibleUp build without the footer text, and I would be extremely grateful for your support in this regard.

Your expertise and support mean a lot to me. Thank you once again for being so helpful!

**Reply by Bukunmikuti** [reply]
You are welcome. I just sent the script to your mail. 
There is a much simpler way to use the script without creating a plugin yourself. You can simply install [WPCode](https://wordpress.org/plugins/insert-headers-and-footers/) plugin and import the script in the dashboard.

**Reply by ManojMk01** [reply]
Could you please provide a more detailed explanation of how this will work? Alternatively, I would greatly appreciate it if you could send me the .zip file of the plugin with the popup footer text changed from 'BibleUp 📖💡' to 'LIB ❤️.' This would allow me to easily import the plugin into WordPress.

**Reply by Bukunmikuti** [reply]
I can't directly modify the plugin to do that. The plugin itself relies on the original script I sent, but it is served through a CDN rather than being bundled with the zip.

To make it simpler, follow these steps to modify the plugin for your use case:
1. Copy the entire text on this page https://cdn.jsdelivr.net/npm/@bibleup/bibleup@1.0.4/dist/umd/bibleup.min.js
2. Create a new file in your system and paste the copied text. You can name the file 'bibleup.min.js.' 
3. Search for the word "BibleUp 📖💡" and replace all to "LIB ❤️" inside the file

To use this custom script in your WordPress site. Follow these steps:

1. Install and activate this WordPress plugin [WPCode](https://wordpress.org/plugins/insert-headers-and-footers/#can%20i%20use%20wpcode%20-%20insert%20headers%20and%20footers%20for%20google%20adsense%3F)
2. Go to Settings Menu and clicking on the Headers & Footers mode.
3. Copy the entire content of the new bibleup.min.js file you modified to LIB ❤️ and paste it inside the **body** text area
4. Leave a space and add the following script under the pasted text
``` js
let page = new BibleUp(document.body, {
popup: 'classic', 
version: 'KJV',
})
page.create()
```
5. Save the changes and reload your WordPress site. It should work fine. 

If you need to change the styling of the popup you can use the [BibleUp Editor](https://bibleup.netlify.app/demo/editor) to style to your taste. Copy the generated options and paste inside the BibleUp instance after `document.body`

For example:
```js
let page = new BibleUp(document.body,
//paste the generated option here
)
page.create()
```

Let me know if this works.
